packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -655,6 +712,7 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions) {
 
   return {
     slots: computedAvailableSlots,
+    teamMember,
```

Comment:
Return the team member email to eventually send to `handleNewBooking`

Reply:
can we make that name clearer, something like `crmContactOwner`?